src/cbmc/cbmc_solvers.cpp

```diff
@@ -235,20 +239,38 @@ std::unique_ptr<cbmc_solverst::solvert> cbmc_solverst::get_smt2(
 
 void cbmc_solverst::no_beautification()
 {
-  if(options.get_bool_option("beautify"))
+  bool beautify = options.get_bool_option("beautify");
```

Comment:
Why is this temporary introduced?

Reply:
I remember I had a reason in the beginning but I may have refactored it enough that it's now unneeded.